### PR TITLE
December 2019 release

### PR DESCRIPTION
**Pull request by bryce-anderson**

### CODE REVIEW

**Comment by bryce-anderson** [comment]
### Testing
This is what the blog post looks like from the locally rendered version:

image
image
image
image
image

File: source/blog/2019-12-13-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+-   finatra-http: Fixed issue in the DefaultMessageBodyReaderImpl that determines if the incoming
+    message is "json encoded". [c1f1a093](https://github.com/twitter/finatra/commit/c1f1a093762fff74f83db8c3adbdf6cf0eb2494a)
+-   inject-modules: Removed the extra registration for closing a client, which used to log false
+    warnings when startup a ClientModule. Only register close after materialized clients.
+    [ddda0b12](https://github.com/twitter/finatra/commit/ddda0b12ce6be14d6832e800764da190ca7f5745)
+-   inject-server: Addressed a race condition that could allow for an AdminHttpServer to be
+    started, even when the disableAdminHttpServer property was set. The AdminHttpServer will
+    no longer start prior to the warm-up phase if disabled. The disableAdminHttpServer property
+    has also been moved to com.twitter.server.AdminHttpServer. PHAB\_ID=D397925\`
```
Comment (yufangong) [inline source/blog/2019-12-13-release-notes.md]: oh here

Comment (mgodave) [inline source/blog/2019-12-13-release-notes.md]: PHAB_ID needs to be manually updated

Comment (enbnt) [reply source/blog/2019-12-13-release-notes.md]: good catch!

**Comment by bryce-anderson** [comment]
@mgodave, good eye. Fixed.

**Pull request merged**